src/v_video.c

```diff
@@ -284,7 +284,7 @@ void WriteGeneratedLumpWad(const char *filename)
 
 video_t video;
 
-#define WIDE_SCREENWIDTH 576 // corresponds to 2.4 aspect ratio
+#define WIDE_SCREENWIDTH 864 // corresponds to 3.6 aspect ratio
```

Comment:
I'm surprised it works, I think there were rendering artefacts. We are currently allocating the video buffer at native resolution (`CreateSurfaces` function in `i_video.c`), perhaps we should allow higher resolutions ("downsampling").

Reply:
If 320px horizonal with 240px vertical corresponds to a 90° FOV then we are slowly approaching a full circle with this. 😁

Reply:
Caught some ASAN crashes related to the status bar background at 32:9, so I need to look at it later:
```
==4248==ERROR: AddressSanitizer: heap-buffer-overflow on address 0x12a314e6d828
at pc 0x7ffa2c71dc25 bp 0x004006deef90 sp 0x004006deef98
WRITE of size 1 at 0x12a314e6d828 thread T0
    #0 0x7ffa2c71dc24 in V_TileBlock64 src\v_video.c:856
    #1 0x7ffa2c70dcb3 in ST_refreshBackground src\st_stuff.c:399
    #2 0x7ffa2c70e3c2 in ST_Drawer src\st_stuff.c:979
    #3 0x7ffa2c59b97d in D_Display src\d_main.c:279
    #4 0x7ffa2c5940a5 in D_DoomMain src\d_main.c:2903
    #5 0x7ffa2c5cdaab in Woof_Main src\i_main.c:65
    #6 0x7ff7721166bd in SDL_main win32\win_launcher.c:36
```

Reply:
I think this crash is related to the video buffer size (`CreateSurfaces` function). I get `video.width > native_width` and crash when switching to 32:9 in the menu.

Reply:
I tried a few things, but I think I'm stuck. There's a temporary workaround here: https://github.com/fabiangreffrath/woof/pull/1364/commits/e6bcd94d5fb122a4e5f6591d00872b44f352a596

Reply:
I think we should re-create surfaces (video buffers) with size greater than native resolution. But I'm thinking, is it practical? Who wants 32:9 without ultra wide display.

Reply:
> Who wants 32:9 without ultra wide display.

Sorry if I'm misunderstanding something, but I have a 16:9 display and I sometimes run the game at 21:9 to take wide screenshots.

Reply:
But at what resolution do you want to take a 21:9 screenshot? And do we need to create a menu item and switch it on the fly, or is the config enough?

@ceski-1 My suggestion is to revert the last 4 commits and make the aspect selection only in config for now, maybe then it will work. I'll do the menu option later.

Reply:
> But at what resolution do you want to take a 21:9 screenshot?

It depends. Sometimes I run Nugget windowed at a resolution greater than my display's to take high-resolution screenshots, such as [this one](https://static.doomworld.com/monthly_2023_08/planisf2.png.63adb3ace98214d14331330d8a266409.png) (which also happens to be 21:9).

Reply:
> @ceski-1 My suggestion is to revert the last 4 commits and make the aspect selection only in config for now, maybe then it will work. I'll do the menu option later.

Okay done. Leaving [this revert](https://github.com/fabiangreffrath/woof/pull/1364/commits/1400451e3ae7f92e11da0b2047359201442fb780) in the commit history in case it's needed later. At 32:9 with "double" resolution, there are garbage pixels at the edges unless the `finetangent` table is used.

Reply:
Okay, the FOV feature is working fine, and the aspect ratios seem to work fine as config only for now. Let's consider in-game aspect ratio selection as out of scope for this PR since it requires reinitialization of the surface buffers and I'm not comfortable working with that code.

Reply:
Works great, thank you. I plan to make a resolution slider where the user can set the maximum height for DRS. Then the width will be calculated based on the current aspect ratio, and video buffers will be recreated if needed.